syntax_and_semantics/literals/command.md

```diff
@@ -0,0 +1,37 @@
+# Command literal
+
+A command literal is a string delimited by backticks `` ` `` or a `%x` percent literal.
+It will be substituted at runtime by the captured output from executing the string in a subshell.
+
+The same [escaping](./string.md#Excaping) and [interpolation rules](./string.md#Interpolation) apply as for regular strings.
+
+Similar to percent string literals, valid delimters fo `%x` are parenthesis `()`, square brackets `[]`, curly braces `{}`, angles `<>` and pipes `||`. Except for the pipes, all delimiters can be nested meaning a start delimiter inside the string escapes the next end delimiter.
+
+The special variable `$?` holds the exit status of the command as a [`Process::Status`](https://crystal-lang.org/api/0.27.0/Process/Status.html). It is only available in the same scope as the command literal.
+
+```cr
+`echo foo`  # => "foo"
+$?.success? # => true
+```
+
+Internally, the compiler rewrites command literals to calls to the top-level method [``.`()``](https://crystal-lang.org/api/latest/toplevel.html#%60(command):String-class-method) with a string literal containing the command as argument: `` `echo #{argument}` `` and `%x(echo #{argument})` are rewritten to `` `("echo #{argument}")``.
```

Comment:
The dot before `` `()`` seems rather confusing to me. I think either `.` or `#` should only be used for instance or class methods of a class/struct/module etc.
You don't call this method on any class or an instance so I don't think a top-level method shouldn't have either `.` or `#`.
```suggestion
Internally, the compiler rewrites command literals to calls to the top-level method [`` `()``](https://crystal-lang.org/api/latest/toplevel.html#%60(command):String-class-method) with a string literal containing the command as argument: `` `echo #{argument}` `` and `%x(echo #{argument})` are rewritten to `` `("echo #{argument}")``.
```
A top-level method is somewhere in the middle of an instance or class method to me.